asserts/snapasserts/validation_sets.go

```diff
@@ -367,7 +386,7 @@ func (v *ValidationSets) CheckInstalledSnaps(snaps []*InstalledSnap, ignoreValid
 
```

Comment:
we generally don't call this on conflicting ValidationSets, is that right?

Reply:
Yes, correct.